tarodb/sqlite/migrations/000007_chain_txns_height_hint.down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE chain_txns
+    DROP COLUMN height_hint;
```

Comment:
However do we need to store this field in this particular table? Alternatively, we can store it in a more context specific place. For example within the minting batch, or an actual log parcel. 

At times we insert this transaction just due to a proof file import (the new anchor tx) as an example).

Reply:
Storing it in `chain_txns` since all transactions that go on-chain should have some context around where to find them (for already confirmed transactions we can set `height_hint= blockheight`)